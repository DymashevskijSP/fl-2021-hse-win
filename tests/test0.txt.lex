LexToken(RULENAME,'A',1,0)
LexToken(EQ,'=',1,1)
LexToken(PLAINTEXT,'B',1,2)
LexToken(ALT,'|',1,5)
LexToken(PLAINTEXT,'C',1,6)
