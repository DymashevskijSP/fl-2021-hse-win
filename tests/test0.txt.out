Node:   A =  B | C: Children  A =  B | C |
Node:  A =  B | C: Children A | = |  B | C |
Node: A: Children EMPTY
Node: =: Children EMPTY
Node:  B | C: Children B | | | C |
Node: B: Children EMPTY
Node: |: Children EMPTY
Node: C: Children EMPTY